app.py

```diff
@@ -1,12 +1,33 @@
 from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+from session_items import get_items, get_item, add_item, save_item
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return renderIndex()
```

Comment:
Why not just move `render_index()` into this method?

Remember that you can use `redirect("/")`  for the other routes.